controller-manifests/cluster-logging.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,361 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: clusterlogging.v0.0.1
```

Comment:
@jwforres would you expect this to v4.0.0 for now?  i'm less clear on the long term plan for these names/versions.

Reply:
updated to 4.1.0